app/src/main/java/com/kickstarter/services/DiscoveryParams.java

```diff
@@ -451,6 +452,15 @@ public boolean shouldIncludeFeatured() {
     }
   }
 
+  /**
+   * Determines if params are for All Projects ("Everything"), i.e. discovery without params.
```

Comment:
In `filterString`, do we need to change `R.string.discovery_everything` to `R.string.All_projects` or something?